### PR TITLE
winscan2pdf: Add version 5.15

### PR DESCRIPTION
**Pull request by worldadventurer**
Another great freeware utility from SoftwareOK. I use this one to scan documents easily & quickly.

### CODE REVIEW

File: bucket/winscan2pdf.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+    "version": "5.15",
+    "description": "Tiny utility to scan documents directly to PDF using any scanner",
+    "homepage": "http://www.softwareok.com/?seite=Microsoft/WinScan2PDF",
```
Comment (linsui) [inline bucket/winscan2pdf.json]:
```suggestion
    "homepage": "https://www.softwareok.com/?seite=Microsoft/WinScan2PDF",
```

Comment (worldadventurer) [reply bucket/winscan2pdf.json]: Good point, @linsui ! I just fixed this pull and also pull #3456. Thanks.

File: bucket/winscan2pdf.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+    "version": "5.15",
+    "description": "Tiny utility to scan docs to PDF using any scanner",
```
Comment (Ash258) [inline bucket/winscan2pdf.json]:
```suggestion
    "description": "Documents to PDF scanner",
```

File: bucket/winscan2pdf.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "homepage": "https://www.softwareok.com/?seite=Microsoft/WinScan2PDF",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.softwareok.com/?seite=Microsoft/WinScan2PDF/Eula"
+    },
+    "url": "https://www.softwareok.com/Download/WinScan2PDF_Portable.zip",
+    "hash": "a184df3eb74f5f13cf79c260b26ac9f6e9affc810af65905c0aa4c9827d2e918",
+    "bin": [
```
Comment (Ash258) [inline bucket/winscan2pdf.json]:
```suggestion
    "pre_install": "if (-not (Test-Path \"$persist_dir\\WinScan2PDF_p.ini\")) { New-Item \"$dir\\WinScan2PDF_p.ini\" | Out-Null }",
    "bin": [
```

File: bucket/winscan2pdf.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+            "WinScan2PDF"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "WinScan2PDF_p.exe",
+            "WinScan2PDF"
+        ]
+    ],
+    "pre_install": "if (-not (Test-Path \"$persist_dir\\WinScan2PDF_p.ini\")) { New-Item \"$dir\\WinScan2PDF_p.ini\" | Out-Null }",
```
Comment (Ash258) [inline bucket/winscan2pdf.json]:
```suggestion
```

Comment (worldadventurer) [reply bucket/winscan2pdf.json]: ok, done

**Pull request merged**